utils/sanitize-utils.ts

```diff
@@ -0,0 +1,9 @@
+import type { Article } from '@prisma/client';
+import Xss from 'xss';
+
+export function addSanitizedDescriptionToArticle<T extends Article>(article: T) {
```

Comment:
Brakuje jeszcze odkodowania encji no a poza tym ta konfiguracja chyba przepuszcza dowolne elementy HTML w description?
https://github.com/typeofweb/polskifrontend/blob/a6da80f77cb637cd479a0cd4a4a676d070f9e67b/utils/excerpt-utils.ts#L18

Reply:
Nie przepuszcza skryptów i o to nam chodzi. Elementy HTML może przepuszczać - wtedy ładniej wygląda to w przypadku opisu np. Require

Reply:
A jeśli ktoś w opisie da `<div style="position: absolute; top: 0; left: 0; right: 0; bottom: 0; background: red;"></div>` ?